### PR TITLE
Milestone 1: Project Proposal

### PR DESCRIPTION
**Pull request by alexanderjshapiro**
You will be creating a website where sellers can post items to sell. You will be writing the requirements for your project in this milestone. You will be able to choose the requirements you want to implement. For example, you can decide if you will allow products to be sold for a fixed price or if you want to allow offers/counteroffers and bidding.

### CODE REVIEW

File: REQUIREMENTS.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+	- **Primary Sequence:** 
+	  1. User is prompted to verify their bidding via a "Proceed" button
+	  2. User clicks on button
+
+	- **Primary Postconditions:** Website displays a message saying "Bidding Received"
+
+	- **Alternate Sequence:**
+	  1. The user doesn't have a payment method saved
+	  2. User is prompted to update their payment method
```
Comment (alexanderjshapiro) [inline REQUIREMENTS.md:142]: It is customary to put a new line (blank line) at the end of every file
```suggestion
	  2. User is prompted to update their payment method

```

File: REQUIREMENTS.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+	  1. User does not input rating
+	  2. User is prompted to input rating or to go back to listing
+	
+	- **Primary Postconditions:** The user is allowed to continue creating their listing 
+	
+	- **Alternate Sequence:**
+	  1. The user enters invalid input into the duration field
+	  2. The user is prompted to edit the duration
+
+5. The user can open up more information on a current listing
```
Comment (alexanderjshapiro) [inline REQUIREMENTS.md:116]:
```suggestion
7. The user can open up more information on a current listing
```

File: REQUIREMENTS.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+	- **Primary Sequence:**
+	  1. A box opens up with any extra information or details the seller wanted to provide
+
+	- **Primary Postcondition:**  Users can see additional information the seller gave
+	
+	- **Alternate Sequence:**
+	  1. No extra information was given, so the box is empty
+
+6.  The user verifies their bidding on a listing
```
Comment (alexanderjshapiro) [inline REQUIREMENTS.md:129]:
```suggestion
8.  The user verifies their bidding on a listing
```

File: REQUIREMENTS.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+	 1. User/Chicken listing is prompted for a rating 
+	 2. Text box for optional input if leaving review
+	 
+	- **Primary Postconditions:** Users able to see and input reviews/ratings for user profiles and chicken listings 
+	
+	- **Alternate Sequence:** 
+	  1. User does not input rating
+	  2. User is prompted to input rating or to go back to listing
+	
+	- **Primary Postconditions:** The user is allowed to continue creating their listing
```
Comment (alexanderjshapiro) [inline REQUIREMENTS.md:110]: Merge error

**Review by pastoraljordan — COMMENTED** [review]
did another request

**Review by davisto7 — APPROVED** [review]

**Review by chels-c — APPROVED** [review]

**Review by pastoraljordan — APPROVED** [review]

**Pull request merged**